devimint/src/devfed.rs

```diff
@@ -247,21 +281,39 @@ impl DevJitFed {
                     let lnd = lnd.get_try().await?.deref().clone();
                     let cln = cln.get_try().await?.deref().clone();
 
+                    debug!(target: LOG_DEVIMINT, "Opening channels between cln and lnd...");
```

Comment:
There's already one inside, generic over node names, etc.

Reply:
Inside what? Are you referring to `open_channels_between_gateways()`? If so, we can't call that here because we no longer have a CLN gateway. `open_channel()` specifically takes an instance of `Lightningd` and `Lnd` types.